src/classes/EventHandlers.js

```diff
@@ -378,15 +378,21 @@ class EventHandlers {
 		let tab_host;
 		let fromRedirect;
 		let block;
+		let ss_unblock;
 		if (bug_id) {
 			app_id = bugDb.db.bugs[bug_id].aid;
 			cat_id = bugDb.db.apps[app_id].cat;
 			incognito = tabInfo.getTabInfo(tab_id, 'incognito');
 			tab_host = tabInfo.getTabInfo(tab_id, 'host');
 			fromRedirect = globals.REDIRECT_MAP.has(request_id);
-			block = this._checkBlocking(app_id, cat_id, tab_id, tab_host, page_url, request_id);
+			const retVal = this._checkBlocking(app_id, cat_id, tab_id, tab_host, page_url, request_id);
```

Comment:
`const {block, ss_unblock} = this._checkBlocking(app_id, cat_id, tab_id, tab_host, page_url, request_id);`

Reply:
The problem with the code `const {block, ss_unblock}` =  is that block should be visible out of scope if(bug_id) { }. We can get away with extra assignment, like:
const {block1, ss_unblock} = ...
block = block1;, but is it better?